### PR TITLE
docs: add a section 'discussion and support'

### PR DESCRIPTION
**Pull request by dmsolr**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [ ] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,26 +185,26 @@ curl --location --request POST 'http://localhost:9195/shenyu/plugin/selectorAndR
...
 
 [![Stargazers over time](https://starchart.cc/apache/incubator-shenyu.svg)](https://starchart.cc/apache/incubator-shenyu.svg)
 
---------------------------------------------------------------------------------  
+---  
+
+# Discussion and Support
+
+* [Mailing Lists](mailto:dev@shenyu.apache.org)
+* QQ Group: 429951241
```
Comment (yu199195) [inline README.md:205]: remove qq group

File: README_CN.md [hunk 2]
```diff
@@ -180,26 +180,26 @@ curl --location --request POST 'http://localhost:9195/shenyu/plugin/selectorAndR
...
 
---------------------------------------------------------------------------------
+---
 
-# Github stars趋势
+# 讨论与支持
 
-[![Stargazers over time](https://starchart.cc/apache/incubator-shenyu.svg)](https://starchart.cc/apache/incubator-shenyu.svg)
+* [邮件列表](mailto:dev@shenyu.apache.org)
+* QQ群：429951241
```
Comment (yu199195) [inline README_CN.md:200]: remove qq group

Comment (dmsolr) [reply README_CN.md:200]: removed

File: README.md (hunk 1 above)
Comment (dmsolr) [reply README.md:205]: removed

Comment (yu199195) [reply README.md:205]: not commit？

Comment (dmsolr) [reply README.md]: Emma, yeah, I don't notice it failed to push.

**Review by yu199195 — APPROVED** [review]

**Pull request merged**